_scripts/_malformeddates.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This script looks portable bourne shell, so we can rely on `/bin/sh` ;-)

```suggestion
#!/bin/sh
```

If we really really want bash, I suggest the following to cope with systems where bash is installed in another directory:

```suggestion
#!/usr/bin/env bash
```